src/features/defi/providers/yearn/components/YearnManager/Deposit/YearnDeposit.tsx

```diff
@@ -630,7 +630,14 @@ export const YearnDeposit = ({ api }: YearnDepositProps) => {
         minWidth={{ base: 'auto', lg: '450px' }}
         maxWidth={{ base: 'auto', lg: '450px' }}
       >
-        {depositRoute && <DefiActionButtons />}
+        {depositRoute && (
+          <DefiActionButtons
+            vaultExpired={
```

Comment:
we should be able to get `expired` key off the vault now with the lib PR correct?

Reply:
no, we will be getting this from https://github.com/shapeshift/lib/pull/346 after it's merged